src/consensus/vote_listener.zig

```diff
@@ -0,0 +1,1463 @@
+const std = @import("std");
+const sig = @import("../sig.zig");
+const vote_parser = @import("vote_parser.zig");
```

Comment:
Please do not import files directly unless it is in/from `sig.zig` or `lib.zig`. This contradicts our standard pattern for imports used throughout the rest of the sig codebase. This should be a `pub const` in lib.zig. It can be imported through `sig.consensus.vote_parser` or you can declare `const consensus = @import("lib.zig");`

Reply:
If this really is the current stance, I'll rather just move the vote_parser code into the vote_listener file.

Reply:
4af0b67c470cde6f76adbdffa841568547d0d206

Reply:
This addresses my initial concern but I'm not sure if it's actually a net improvement. Anyway I don't think this is too important. Let's revisit the API topic when our deadline schedule is a bit more relaxed. I agree that we should constrain our public APIs, but we should agree on a pattern and then implement it throughout the codebase.